### PR TITLE
Fix bug in DevOps logging command

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Use the correct syntax for debug logging in DevOps.

### CODE REVIEW

File: eng/common/scripts/logging.ps1 [hunk 1]
```diff
@@ -31,10 +31,10 @@ function LogDebug
 {
     if ($isDevOpsRun) 
     {
-        Write-Host "##vso[task.LogIssue type=debug;]$args"
+        Write-Host "[debug]$args"
     }
     else 
     {
-        Write-Debug "$args"
+        Write-Debug "$args" -Debug
```
Comment (weshaggard) [inline eng/common/scripts/logging.ps1:38]: Why do you want to always pass `-Debug` here? Don't you want to only have this write if someone passes `-Debug` as a common parameter? https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-7#debug

Comment (chidozieononiwu) [reply eng/common/scripts/logging.ps1:38]: I guess I can remove that. But it very helpful when running the tools locally.

**Review by weshaggard — APPROVED** [review]
The one comment but otherwise looks good.

File: eng/common/scripts/logging.ps1 (hunk 1 above)
Comment (weshaggard) [reply eng/common/scripts/logging.ps1]: It is like Write-Verbose in that you don't want them to be displayed by default and only if you pass in the extra option should they be displayed.

**Pull request merged**